types/User.js

```diff
@@ -5,6 +5,7 @@ const UserId = S.string().maxLength(128)
 const UserInstallToken = S.string().maxLength(128)
 const UserApiKey = S.string().maxLength(128)
 const Username = S.string().minLength(3).maxLength(64).pattern('^\\w+$')
+const IlpPointer = S.string().minLength(3).maxLength(128)
```

Comment:
Since there's a spec for it, we might as well lock it down to just valid pointers:

```(?=^.{1,1023}$)^(g|private|example|peer|self|test[1-3]?|local)([.][a-zA-Z0-9_~-]+)+$```

ref: https://interledger.org/rfcs/0015-ilp-addresses/

Reply:
if i'm reading the spec right... now second guessing if that's a payment pointer or something else. where's kev

Reply:
it's not - the pointer seems to be a $ (optional) plus a URL, so we can figure out validation of it later